lib/route_opaque.h

```diff
@@ -33,6 +33,12 @@ struct bgp_zebra_opaque {
 
 	/* Show at least 10 large-communities AA:BB:CC */
 	char lcommunity[LCOMMUNITY_SIZE * 30];
+
+	/* 32 bytes seems enough because of
+	 * bgp_path_selection_confed_as_path which is
+	 * `Confederation based AS Path`.
+	 */
+	char best_path_reason[32];
```

Comment:
I do see that all the Best Path selections reasons are under 32 for now, But, Can you do a  #define for this max length instead of hard coded 32 ?

Reply:
Fixed.